### ISSUE TITLE
`deno bundle` strips `static` from ES private properties

### ISSUE THREAD

**Issue by ebebbington** [denoland/deno#11287]
```
deno 1.11.3+334cef0 (canary, x86_64-pc-windows-msvc)
v8 9.2.230.12
typescript 4.3.2
```

## Repro

```ts
// tmp.ts
class A {
    static #name = "A"

    someFn() {
        return A.#name
    }
}

new A().someFn()
```

```shell
$ deno bundle tmp.ts tmp.js
$ cat tmp.js
class A {
    #name = "A";
    someFn() {
        return A.#name;
    }
}
new A().someFn();
```

Now the `someFn` method is broken

**Comment by kitsonk** [comment]
This impacts `--no-check` as well.

**Comment by kitsonk** [comment]
Fixed in swc upstream.  Should just need to update.

**Comment by ebebbington** [comment]
great news :)

**Comment by bartlomieju** [comment]
Should have been fixed in #11626
